cve/system/ksp-cve-2021-24867-block-wordpress-supply-chain-attack.yaml

```diff
@@ -17,12 +17,10 @@ spec:
     severity: 6
     matchPatterns:
     - pattern: /**/initial.php
-  action:
-    Block
+    action: Block
```

Comment:
This one is good @afzalbin64, I can merge this if you'll raise PR for each policy separately